### DISCUSSION TITLE
posts/2024/08/luna-keeps-an-eye-on-every-soul/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#61] [General]
# posts/2024/08/luna-keeps-an-eye-on-every-soul/

https://guozheng.rbind.io/posts/2024/08/luna-keeps-an-eye-on-every-soul/

**Comment by earfanfan** [comment]
啊，要是我现在还能拥有如此敏锐的感知能力，一定会偷着乐。说实话，我觉得你年纪轻轻文笔就比我好好多哦，也许随着时间推移，笔力还会往不同方向提升的，未来可期呀，小伙子诶。

**Reply by residualsun1** [reply]
没想到你会这么想！这次我不按常理出牌，我要~厚着脸皮~接受你的肯定😄（其实早上就看到了——看完之后整个人就一下子精神了，乐了好久，但一下子又不知道怎么回复OwO）。不过不过，不当说我的文笔比你更好，话说回头，你的[“生活”](https://yuanfan.rbind.io/tags/%E7%94%9F%E6%B4%BB/)类标签博客可才是敏锐感知力的典范呀！

最后，想了一下，不管是不是奢望，希望我们都会一直一直写下去。
